Fix issues with Compiling docker

frontend_1  | ./src/ui/Select/Select.tsx
frontend_1  | Module not found: Can't resolve 'awesome-debounce-promise' in '/app/src/ui/Select'

Working through the issue with the docker deployment to determine why the build process is not resolving this issue. Will fix before we refresh an aws environment.

--- a/frontend/docker/Dockerfile
+++ b/frontend/docker/Dockerfile
@@ -1,4 +1,4 @@
-FROM node:14.15-alpine as debug
+FROM node:12-alpine as debug
  
 RUN apk add --no-cache g++ make python
 
@@ -10,6 +10,8 @@ COPY package*.json ./
  
 # Installs all node packages
 RUN npm install
+# In windows, this may be needed.
+# RUN npm install awesome-debounce-promise
  
 # Copies everything over to Docker environment
 COPY . .

